vlog4j-parser/src/test/java/org/semanticweb/vlog4j/syntax/parser/RuleParserTest.java

```diff
@@ -411,4 +412,24 @@ public void testBlankPredicateName() throws ParsingException {
 		RuleParser.parse(input);
 	}
 
+	@Test
+	public void DatatypeConstantgRoundTripTest() throws ParsingException {
```

Comment:
This test does not do roundtrip.

Reply:
what do you suggest?, should we just remove all of them and the ones in the entitytest are enough ?